R/add_keep_population_flag.R

```diff
@@ -0,0 +1,146 @@
+#' Add keep_popluation flag
+#'
+#' @description Add keep_population flag to individual files
+#' @param data A data frame
+#'
```

Comment:
Could you update the documentation please and add individual_file and year as parameters. Thanks